utils/serialise.js

```diff
@@ -1,12 +1,13 @@
 module.exports = serialise
 
-function serialise (type, attributes, id) {
+function serialise (type, attributes, id, createdAt) {
   if (!type || !attributes) {
     throw new Error('Serialisation must include a type and some attributes.')
   }
   var data = {
     type: type,
-    attributes: attributes
+    attributes: attributes,
+    createdAt: createdAt
   }
```

Comment:
I think you might need to add `createdAt` next to `username` and `password` for the two `t.deepEqual` calls